src/sentry/replays/post_process.py

```diff
@@ -153,7 +231,7 @@ def _archived_row(replay_id: str, project_id: int) -> dict[str, Any]:
 }
 
 
-def extract_click_fields(item: MutableMapping[str, Any]) -> list[dict[str, Any]]:
+def extract_click_fields(item: MutableMapping[str, Any]) -> List[Dict[str, Any]]:
```

Comment:
i wonder if we could use TypedDicts here as well 🤔

Reply:
discussed offline